src/NzbDrone.Api/Profiles/LanguageValidator.cs

```diff
@@ -0,0 +1,45 @@
+﻿
+using System.Collections.Generic;
+using System.Linq;
+using FluentValidation;
+using FluentValidation.Validators;
+using NzbDrone.Api.Profiles;
+
+namespace NzbDrone.Core.Validation
+{
+    public static class LanguageValidation
+    {
+        public static IRuleBuilderOptions<T, IList<ProfileLanguageItemResource>> MustHaveAllowedLanguage<T>(this IRuleBuilder<T, IList<ProfileLanguageItemResource>> ruleBuilder)
```

Comment:
`MustHaveAllowedLanguage` sounds a bit awkward to me, makes it sounds like we're testing a specification. Thoughts?